ci/supportfiles/.sparkSettings.json

```diff
@@ -0,0 +1 @@
+{"useSparkPermissionsForPxFileAccess" : true}
```

Comment:
There shouldn't be any spark settings file by default.  This setting file should only be used to override default setting values.  Please remove this file.